examples/allennlp/classifier.jsonnet

```diff
@@ -0,0 +1,72 @@
+// Paths to data.
+
+// use dev data for training due to time consumption
+local TRAIN_PATH =  "https://s3-us-west-2.amazonaws.com/allennlp/datasets/imdb/dev.jsonl";
```

Comment:
Use `DEV_DATA` for training to reduce training time.

Reply:
Sorry for confusing you.
I note that this example uses `dev.jsonl` for training in 1bc53db.